Automattic/Gravatar-SDK-Android
# `gravatar-ui`- Screenshot testing - Roborazzi

Closes #114 

### Description

This PR adds screenshot testing using [Roborazzi](https://github.com/takahirom/roborazzi). At the moment, screenshots have been added to the `gravatar-ui` module to verify that we don't break anything between iterations of the UI components.

If you execute the verification task, you'll find the report in:

`gravatar-ui/build/reports/roborazzi/index.html`

**Note:** We should add a placeholder to the `Avatar` component in order to prevent it from showing nothing. Another option could be adding the skeleton while the image is being loaded. I've tried but encountered some issues with the screenshots, so I'll leave it for another PR.

### Testing Steps

- Review CI and verify tests have been executed
- Execute locally the `record` and `verify` commands:
```
- Record -> ./gradlew recordRoborazziDebug
- Verify -> ./gradlew verifyRoborazziDebug
```
- Play with the test, trying to break something on purpose to verify it works.
